LibGit2/Functions/Test-GitBranch.ps1

```diff
@@ -0,0 +1,54 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function Test-GitBranch
+{
+    <#
+    .SYNOPSIS
+
+    Checks if a branch exists.
+
+    .DESCRIPTION
+
+    Returns true if branch exists.  False otherwise.

```

Comment:
"The `Test-GitBranch` command tests if a branch exists in a Git repository. It returns `$true` if a branch exists; `$false` otherwise. Pass the branch name to test to the `Name` parameter.